### PR TITLE
fix: Fix no project name and mirror name when creating a mirror.

### LINKED ISSUE #472: When creating a service via b2i, upload the war file and return that the current project was not found
**Describe the bug**

image

[POST] https://console.kubesphere.io/apis/devops.kubesphere.io/v1alpha1/namespaces/project001/s2ibinaries/b2i-war-java8-war7zpa

**Versions used(KubeSphere/Kubernetes)**
KubeSphere: 3.0.0(2020-06-22)

/assign @shaowenchen

### PR DESCRIPTION
**Pull request by harrisonliu5**
**What type of PR is this?**
 /kind bug

**What this PR does / why we need it**:
 Fix no project name and mirror name when creating a mirror.

**Which issue(s) this PR fixes**:
Fixes #472

### CODE REVIEW

File: src/actions/imagebuilder.js [hunk 1]
```diff
@@ -16,30 +16,25 @@
...
 import formPersist from 'utils/form.persist'
 import FORM_STEPS from 'configs/steps/imagebuilder'
 
 export default {
   'imagebuilder.create': {
     on({ store, cluster, namespace, module, success, ...props }) {
-      const kind = MODULE_KIND_MAP[module]
       const formTemplate = {
-        [kind]: FORM_TEMPLATES[module]({
+        ...FORM_TEMPLATES[module]({
```
Comment (leoendless) [inline src/actions/imagebuilder.js:31]: no need to spread the result

File: src/components/Base/Upload/utils/request.js [hunk 2]
```diff
@@ -16,6 +16,8 @@
  * along with KubeSphere Console.  If not, see <https://www.gnu.org/licenses/>.
  */
 
+const { getClusterUrl } = require('utils/index')
```
Comment (leoendless) [inline src/components/Base/Upload/utils/request.js:19]: use import instead ?

File: src/actions/imagebuilder.js (hunk 1 above)
Comment (harrisonliu5) [reply src/actions/imagebuilder.js]: solved

File: src/components/Base/Upload/utils/request.js (hunk 2 above)
Comment (harrisonliu5) [reply src/components/Base/Upload/utils/request.js]: solved

**Pull request merged**